### ISSUE TITLE
Inconsistent Line endings

### ISSUE THREAD

**Issue by dmgdotnet** [otykier/TabularEditor#133]
There seems to be an inconsistency with line endings and the 'Save to folder' feature.  There are certain scenarios where the endings will be '\r\n' and other where they are '\n'.

Our workflow is:
1. Open from folder after pull from master branch
2. Deploy to local database
3. Open from database
4. Make change to 1 measure
    [Total Pipeline Eur]*[Selected Rate]
    _changed to_
    [Total Pipeline Eur]*[Selected Rate]*-1
5. Save to database
6. Save to folder

We then see changes to 24 measures in total from multiple tables which don't seem to be related.  The changes in the files are '\n' is replaced with '\r\n'.

"expression": "\nIF
to
"expression": "\r\nIF

Next we:
1. Open from same folder that now has \r\n
2. Deploy to local database
3. Open from database
4. Save to folder without any changes to measures

And the line endings are saved back to '\n' - let me know if this is hard to follow and I'll try to clean up the explanation.

PS, any tips on a better workflow appreciated :)

**Comment by dmgdotnet** [comment]
I've been trying to track this down with repro scenarios and some testing with TOM and JSON.NET and I can't reproduce outside Tabular Editor.  The encoded line endings seem to only be added to the Tabuler Editor Model after the Save to Database happens. 

Tests (initial model.bim file has now \r inside file):

1. Open Model.bim , Save AS new Model.bim = No \r

**Comment by dmgdotnet** [comment]
Sounds great! Just for clarity though, I may have confused the situation a little with my poor use of terminology.  It's not the actual file line endings I'm having issues with, it is the encoded line endings in the expression and annotation text. Sorry if i wasn't clear on that!  Here is an example, red is the issue and green is the original before the db save.

![image](https://user-images.githubusercontent.com/2540043/37024316-3089c614-2174-11e8-8483-6bede3c27183.png)

**Comment by otykier** [comment]
Yes, this is also what I meant :-)
